share/patch/patch_db_zonemaster_backend_ver_11.1.0.pl

```diff
@@ -178,61 +282,23 @@ sub patch_db_postgresql {
     try {
         $db->create_schema();
 
-        print( "\n-> (1/3) Populating new result_entries table\n" );
-
-        $dbh->do(q[
-            INSERT INTO result_entries (
-                hash_id, args, module, level, tag, timestamp, testcase
-            )
-            (
-                SELECT
-                    test_results.hash_id,
-                    COALESCE(res->'args', '{}') AS args,
-                    CASE res->>'module'
-                        WHEN 'DNSSEC' THEN res->>'module'
-                        ELSE initcap(res->>'module')
-                      END AS module,
-                    log_level.value AS level,
-                    res->>'tag' AS tag,
-                    (res->>'timestamp')::real AS timestamp,
-                    CASE
-                        WHEN res->>'testcase' IS NULL THEN ''
-                        WHEN res->>'testcase' LIKE 'DNSSEC%' THEN res->>'testcase'
-                        ELSE initcap(res->>'testcase')
-                      END AS testcase
-                FROM test_results,
-                     json_array_elements(results) as res
-                     LEFT JOIN log_level ON (res->>'level' = log_level.level)
-            )
-        ]);
+        # Make sure the planner knows that log_level is a small table
+        # so it can optimize step 1 appropriately
+        $dbh->do(q[ANALYZE log_level]);
+
+        print( "\n-> (1/2) Populating new result_entries table\n" );
+        _patch_db_postgresql_step1( $dbh );
```

Comment:
Only one parameter in the call here. Shouldn't the default value of the second parameter be set here?